src/ensembl/src/global/globalHelper.ts

```diff
@@ -1,9 +1,11 @@
 import { BreakpointWidth } from './globalConfig';
 
-export const getBreakpoint = (width: number): BreakpointWidth => {
-  if (width > BreakpointWidth.DESKTOP) {
+export const getBreakpoint = (
+  key: keyof typeof BreakpointWidth
+): BreakpointWidth => {
+  if (BreakpointWidth[key] >= BreakpointWidth.DESKTOP) {
     return BreakpointWidth.DESKTOP;
-  } else if (width > BreakpointWidth.LAPTOP) {
+  } else if (BreakpointWidth[key] >= BreakpointWidth.LAPTOP) {
```

Comment:
`PHONE` and `BIG_DESKTOP` also need to be here

Reply:
You have them in your branch ;-) Perhaps add them when you merge this PR/dev into your branch?